app/Services/SubscriberService.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace App\Services;
+
+
+use App\Subscriber;
+use App\Tools\ResponseTool;
+
+class SubscriberService
+{
+    public static function subscribe($email)
+    {
+        $checkEmail = Subscriber::whereEmail($email)->count();
+        if ($checkEmail == 0) {
+            Subscriber::create([
```

Comment:
if there is only 1 item, make it one line
```suggestion
            Subscriber::create(['email' => $email]);
```

Reply:
Ok, I see 😊